test/core/security/tls_security_connector_test.cc

```diff
@@ -1089,6 +1127,38 @@ TEST_F(TlsSecurityConnectorTest,
   core_external_verifier->Unref();
 }
 
+TEST_F(TlsSecurityConnectorTest,
```

Comment:
Similar comments here as above.

Reply:
Addressed in [this comment chain](https://github.com/grpc/grpc/pull/32335#discussion_r1103285702)